src/main/java/TreeHandlers/TreeHandler.java

```diff
@@ -0,0 +1,80 @@
+package TreeHandlers;
+
+import Entities.*;
+import Interactors.GameLogic;
+
+import java.util.HashMap;
+
+public class TreeHandler {
+    GameLogic gameLogicInteractor;
+    Player currentPlayer;
+    Board board;
+    HashMap<String, Integer> selectedOptions = new HashMap<String, Integer>();
+    int returnPlayerIndex = -1;
+    Player[] players;
```

Comment:
this should be an ArrayList in case we have to remove a Player from the players array (ex. Bankruptcy)

Reply:
Yes, I will change the players array to an arraylist soon!